sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -670,10 +686,13 @@ class MapboxMap :
    * Note that if the given `camera` shows the antimeridian, the returned wrapped [CoordinateBounds]
    * might not represent the minimum bounding box.
    *
+   * This API isn't supported by Globe projection and will return a no-op result.
```

Comment:
Shall we clarify no-op result? in this case this function will return a CoordinateBounds, is it nullable result or it's a CoordinateBounds with meaningless value?